packages/core/src/config/snapwp-config-manager.ts

```diff
@@ -374,7 +374,8 @@ class SnapWPConfigManager {
 	 */
 	static getGraphqlUrl(): string {
 		return generateGraphqlUrl(
-			SnapWPConfigManager.getConfig().wpHomeUrl,
+			SnapWPConfigManager.getConfig().wpSiteUrl ||
+				SnapWPConfigManager.getConfig().wpHomeUrl,
```

Comment:
wpSiteUrl already falls back to `wpHomeUrl`.
```suggestion
			SnapWPConfigManager.getConfig().wpSiteUrl,
```